completions/ssh

```diff
@@ -451,20 +454,25 @@ _comp_xfunc_ssh_scp_remote_files()
         path=$(ssh -o 'Batchmode yes' "$userhost" pwd 2>/dev/null)
     fi
 
+    local escape_replacement='\\\\\\&'
+    if [[ $no_overescape == "no-overescape" ]]; then
```

Comment:
Errors in `nounset mode`, unnecessary quoting. (A different fix likely appropriate if/when we tweak how the parameter is handled.)

```suggestion
    if [[ ${no_overescape-} == no-overescape ]]; then
```